internal/services/mssql/mssql_database_resource.go

```diff
@@ -558,6 +581,22 @@ func resourceMsSqlDatabaseCreateUpdate(d *pluginsdk.ResourceData, meta interface
 		return fmt.Errorf("waiting for create/update of %s: %+v", id, err)
 	}
 
+	if features.ThreePointOh() {
+		statusProperty := sql.TransparentDataEncryptionStatusDisabled
+		encryptionStatus := d.Get("transparent_data_encryption_enabled").(bool)
+		if encryptionStatus {
+			statusProperty = sql.TransparentDataEncryptionStatusEnabled
+		}
+		_, err := transparentEncryptionClient.CreateOrUpdate(ctx, id.ResourceGroup, id.ServerName, id.Name, sql.TransparentDataEncryption{
```

Comment:
I've confirmed in the SDK, but I'm really surprised this isn't a future - do we need to poll to wait for this to finish replicating at all?

Reply:
We'll now wait until the database has stopped scaling.